src/test/java/org/broadinstitute/hellbender/engine/dataflow/datasources/VariantShardUnitTest.java

```diff
@@ -30,4 +30,10 @@ public void getVariantShardsFromIntervalTest(GATKRead read, Iterable<VariantShar
             List<VariantShard> foundShards = VariantShard.getVariantShardsFromInterval(read);
             Assert.assertEquals(foundShards, expectedShards);
     }
+
+    @Test
+    public void verifyDivisibilityWithRefShard() {
```

Comment:
This test makes sense but it seems funny having it here.  Maybe it should just move into the test data class itself?